examples/usbus_minimal/Makefile

```diff
@@ -16,18 +16,29 @@ USEMODULE += usbus
 USEMODULE += auto_init_usbus
 
 # USB device vendor and product ID
-DEFAULT_VID = 1209
-DEFAULT_PID = 0001
+export DEFAULT_VID = 1209
```

Comment:
Why is it necessary to export these variables?

Reply:
They are being used in `examples/usbus_minimal/Kconfig`